AdminUi/test/AdminUi.Tests.Integration/StepDefinitions/ClientsStepDefinitions.cs

```diff
@@ -1,23 +1,61 @@
-using AdminApi.Tests.Integration.API;
-using AdminApi.Tests.Integration.Extensions;
-using AdminApi.Tests.Integration.Models;
+using AdminUi.Tests.Integration.API;
+using AdminUi.Tests.Integration.Extensions;
+using AdminUi.Tests.Integration.Models;
+using Newtonsoft.Json;
 
-namespace AdminApi.Tests.Integration.StepDefinitions;
+namespace AdminUi.Tests.Integration.StepDefinitions;
 
 [Binding]
 [Scope(Feature = "GET Clients")]
+[Scope(Feature = "DELETE Clients")]
 public class ClientsStepDefinitions : BaseStepDefinitions
 {
     private readonly ClientsApi _clientsApi;
+    private string _clientId;
     private HttpResponse<List<ClientDTO>>? _response;
 
     public ClientsStepDefinitions(ClientsApi clientsApi) : base()
     {
         _clientsApi = clientsApi;
+        _clientId = string.Empty;
+    }
+
+    [Given(@"a Client c")]
+    public async Task GivenAClientC()
+    {
+        var createClientRequest = new CreateClientRequest
+        {
+            ClientId = string.Empty,
+            DisplayName = "a-client-display-name",
+            ClientSecret = string.Empty
```

Comment:
Wait... it is possible to create a client with an empty client id and an empty secret?
We should add some validation to prevent this. I would say that the minimum length of a client id should be 5 and the minimum length of a client secret should be 10 (I know that our `test` client has a secret with a lower length, but we shouldn't reduce common security for the sake of a easy test data. ;-)
(I'm only talking about application data validation; it's not necessary to add the restrictions to the database)

Reply:
It's possible to send the request without specifying a client id and secret because if either is null or empty, then they are automatically generated.